rinja_parser/src/expr.rs

```diff
@@ -325,6 +308,33 @@ impl<'a> Expr<'a> {
     }
 }
 
+fn token_xor(i: &str) -> ParseResult<'_> {
+    let (i, good) = alt((value(true, keyword("xor")), value(false, char('^'))))(i)?;
+    if good {
+        Ok((i, "^"))
+    } else {
+        Err(nom::Err::Failure(ErrorContext::new(
+            "the binary XOR operator is called 'xor' in rinja",
```

Comment:
Just noticed but we use backtick to "highlight" elements like `xor` in the error messages, not quotes. Sorry, nitpicking time. ^^'

Reply:
Nitpicking is half the fun of reviewing. :D Fixed.